magento/pwa-studio
# fix: Cart/Checkout summary uses cache-and-network to fetch latest

## Description

We use the price summary fragment to update it when on the cart or checkout page but we didn't configure the component to update itself when navigating to the page from another page. In this case the fix is pretty simple, we can use `cache-and-network` to indicate that we should use cached data to render but then true-up with the server.

## Related Issue





Closes [PWA-824](https://jira.corp.magento.com/browse/PWA-824).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps
1. Add an item to your cart.
2. Click mini cart and "CHECKOUT" to navigate to /checkout.
3. After page loads, click the back button in your browser, or use the venia logo to navigate to a product page.
4. Add a product to your cart.
5. Click mini cart and "CHECKOUT" to go back to the checkout page. Both total and subtotal should be correct!

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
